### ISSUE TITLE
Migrating PulseEffects Multiband Compressor configuration to EasyEffects Multiband Compressor 

### ISSUE THREAD

**Issue by mike1773** [wwmm/easyeffects#1613]
I have an issue where after importing my old PulseEffects configuration to EasyEffects, the processed sound very different, because EasyEffects has more options for the Multiband Compressor, namely:

-Operating Mode
-Sidechain Boost
-Release Threshold
-Band Sidechain Options: PreAmp, Reactivity, Lookahead

What values should be chosen to resemble the old setup from PulseEffects?

Thank you

PulseEffects:
![pe2](https://user-images.githubusercontent.com/108511830/176949641-46f1269f-8c41-4bbd-9dd6-103462da4f0e.png)

EasyEffects:
![ee2](https://user-images.githubusercontent.com/108511830/176949674-a060e3b7-f627-48a8-8388-35df071354af.png)

**Comment by Digitalone1** [comment]
The best thing you can do is to recalibrate your parameters with the new MBC because the LSP version is different from Calf one.

Compression Mode has to be Downward since Calf does not have the Upward mode. Band Lookahed can stay to 0, otherwise it introduces delay. The Classic Mode is somewhat similar to the Calf version, but in my experience the Modern is better. The Threshold in the Calf version is the Attack Threshold in the LSP one. For difference between Attack and Release Threashold, take a look at the manual.

Anyway, even if you set similar parameters, they can sound different because they are designed differently, so it needs to be recalibrated.

**Comment by Digitalone1** [comment]
If you can run it in carla-rack, you can tweak with the native UI which has a nice spectrum, so you can see the differences with the signal on various options.

If you have Pipewire Jack layer, just run `pw-jack carla-rack` and connect carla rack to the application and the output device (through helvum maybe).

**Comment by mike1773** [comment]
Thank you very much for your detailed responses! Yes, I just noticed that EasyEffects uses Linux Studio Plugins instead of Calf Studio Gears. 

If I set Band Lookahead to 0, the sound feels harsher, and some research on Google seems to confirm this, mentioning that having Lookahead handles spikes and transients better (at the cost of some latency). I'm still not sure what the Reactivity option does, it does not seem to modify the audio too much, at least in my case.

I managed to get carla-rack up and running. I added the old Calf MBC, the new LSP MBC and a LSP spectrum analyzer to be able to do some testing, thanks again for the tip!

In conclusion, is there a possibility to bring back Calf Studio Gears to EasyEffects, since sound is subjective in my opinion? (For example in the bottom right corner to be able to select the provider). If not, I guess this issue can be closed and I'll continue to test and match the sound manually.

Cheers!

**Comment by Digitalone1** [comment]
> In conclusion, is there a possibility to bring back Calf Studio Gears to EasyEffects, since sound is subjective in my opinion? 

No, EasyEffects aims to offer the best plugins and LSP MBC is better because it let the user choose the number of bands, has more options and can do upward compression. 

We are also planning to migrate the Gate plugins to the LSP versions.
